### PR TITLE
dotnet 6

### PR DESCRIPTION
**Pull request by sleepwalkerffs**
Upate for .NET 6. 
I hope I did not oversee any place. :)

### CODE REVIEW

**Comment by gfoidl** [comment]
Thanks for the PR.
This week I'm out of office and mobile only, so I just see that CI fails, but can't tell you why.

Do we need a special target for NET 6?

File: .azure/pipelines/jobs/steps/dotnet-install.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 steps:
+  - task: UseDotNet@2
```
Comment (Ne4to) [inline .azure/pipelines/jobs/steps/dotnet-install.yml:2]: I think these two lines ([1](https://github.com/sleepwalkerffs/trx2junit/blob/dotnet-6-update/.azure/pipelines/jobs/build_and_test.yml#L14), [2](https://github.com/sleepwalkerffs/trx2junit/blob/dotnet-6-update/.azure/pipelines/jobs/e2e_tests.yml#L8)) should be uncommented to make it work

**Comment by Ne4to** [comment]
BTW, before updated version is available you can use following workaround to run it with .NET 6.0
`export DOTNET_ROLL_FORWARD=Major` ([docs](https://docs.microsoft.com/en-us/dotnet/core/versions/selection))

**Comment by chrishaland** [comment]
Came here to add a PR for this, but see you're way ahead of me.

You might consider updating the target framework in `samples/Directory.Build.props` to `net6.0` as well.

**Comment by sleepwalkerffs** [comment]
Unfortunatly i am on a fair until thursday so i'll do it either in between or on thursday, thx for the help and thx for the suggestions.

**Comment by gfoidl** [comment]
https://github.com/gfoidl/trx2junit/pull/89 will supercede this PR.

**Pull request closed without merging**